art_archive_project/api/artists/controllers/artists.py

```diff
@@ -1,10 +1,13 @@
-from flask import url_for
 from flask import request
 
+from api import db
 from api.artists.models import Artist
 from api.artists.controllers import artists_api
 from api.utils.json_decorator import json
+from api.utils.response_wrapper import ok_response, created_response
 from api.utils.errors import unprocessable_entry
+from api.utils.url_helper import get_absolute_url
+from api.utils.nullify import nullify
 
 
 @artists_api.route("/", methods=['GET', 'POST'])
```

Comment:
artists_list function is extremely, what can we do to reduce the mass here? Let's try to aim for each function (including helper functions) not go beyond 18 lines. One suggestion here is to create separate helper functions for GET and POST. What else can we do?